guides/rails_guides.rb

```diff
@@ -26,5 +26,5 @@
   only:      env_value["ONLY"],
   kindle:    env_flag["KINDLE"],
   language:  env_value["GUIDES_LANGUAGE"],
-  direction: env_value["DIRECTION"].to_sym
+  direction: env_value["DIRECTION"]&.to_sym || :ltr
```

Comment:
How about using String to manage `direction` and remove `to_sym` call?  
We treat other env values as String, and it seems that the more consistent it is.